src/api/python/cvc5.pxd

```diff
@@ -27,6 +27,29 @@ cdef extern from "<string>" namespace "std":
         const wchar_t* data() except +
         size_t size() except +
 
+cdef extern from "<tuple>" namespace "std" nogil:
+    cdef cppclass tuple[T, U, S]:
+        ctypedef T first_type
+        ctypedef U second_type
+        ctypedef S third_type
+        T first
+        U second
+        S third
+        tuple() except +
+        tuple(tuple&) except +
+        tuple(T&, U&, S&) except +
+        bint operator==(tuple&, tuple&)
+        bint operator!=(tuple&, tuple&)
+        bint operator<(tuple&, tuple&)
+        bint operator>(tuple&, tuple&)
+        bint operator<=(tuple&, tuple&)
+        bint operator>=(tuple&, tuple&)
```

Comment:
I think we should only "import" what we need. I think we don't need any of this (but not sure)
```suggestion
    cdef cppclass tuple[T, U, S]:
        pass
```

Reply:
Right! Thanks, deleted here: f68979e6b4742f4210c2420af6fff6501061a15d